website/blog/2021-07-27-use-of-plugin-orchestration-in-Apache-APISIX.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "插件编排在 Apache APISIX 中的应用与实践"
+author: juzhiyuan
+authorURL: "https://github.com/juzhiyuan"
+authorImageURL: "https://avatars.githubusercontent.com/u/2106987?v=4"
+---
+> [@juzhiyuan](https://github.com/juzhiyuan), Apache APISIX PMC from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+<!--truncate-->
+
+本文作者琚致远，为 Apache APISIX PMC，在支流科技负责企业产品与大前端技术。通过阅读本文，您可以了解 Apache APISIX 与基本使用场景，以及在低代码潮流下，Apache APISIX 是如何集成“拖拽”的插件编排能力的。
+
+## 什么是 Apache APISIX
+
+Apache APISIX 是一个生产可用的七层全流量处理平台，可作为 API 网关处理业务入口流量，具有极高性能、超低延迟的显著特性。它内置了 50 多种插件，覆盖身份验证、安全防护、流量控制、Serverless、可观测性等多个方面，可满足企业客户常见的使用场景。
+
+如下方架构图所示，Apache APISIX 分为数据面（左侧）与控制面（右侧）两部分：通过控制面下发配置到 ETCD，数据面借助丰富的插件处理内外流量。
+
+![2021-07-27-1](../static/img/blog_img/2021-07-27-1.png)
```

Comment:
`[2021-07-27-1]` this part will be the `alt` for the `<img>` after render the web page, it would be better to define a semantic name for the image, this is friendly for SEO ^_^

Reply:
Good! Let me edit the alt content.

Reply:
same for other image name ^^

Reply:
ok